cfpb/consumerfinance.gov
# TCCP: Add "About us" sidebar, minor content updates

This commit adds an "About us" sidebar to the TCCP landing page and card detail view.

## How to test this PR

After running a local server, visit the landing page at:

http://localhost:8000/consumer-tools/credit-cards/explore-cards/

Click "see all cards" then click on any card to see the detail page.

## Notes and todos

We have a few instances of this hardcoded text in the codebase, plus another instance of it as a Wagtail ReusableText snippet. It'd be nice if those were all unified somehow, but I'm leaving that for a future PR.

## Checklist
